sql/src/main/java/org/apache/druid/sql/calcite/planner/PlannerContext.java

```diff
@@ -142,6 +146,8 @@ public class PlannerContext
   // set of attributes for a SQL statement used in the EXPLAIN PLAN output
   private ExplainAttributes explainAttributes;
   private PlannerLookupCache lookupCache;
+  // Lookup loading spec for a given task
```

Comment:
```suggestion
  /**
   * Lookup loading spec used by MSQ tasks.
   */
```

Reply:
Have removed this comment as per suggestion by Karan as it isn't really adding any additional info.